app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(line: str) -> str:
+    command, old_file, new_file = line.split(" ")[1:]
+
+    if command != "cp" or old_file == new_file:
+        return "Incorrect command or your file names are the same!"
```

Comment:
I'd say that the function shouldn't return this string. You should either use empty `return` statement or raise an error with this message.

Reply:
Thank you!